sigmastate/src/main/scala/sigmastate/crypto/CryptoContext.scala

```diff
@@ -0,0 +1,25 @@
+package sigmastate.crypto
+
+import java.math.BigInteger
+
+/** A context for cryptographic operations. */
+abstract class CryptoContext {
+  /** The order of the underlying finite field. */
+  def getModulus: BigInteger
+
+  /** The order of the underlying group. */
+  def getOrder: BigInteger
```

Comment:
Why getOrder ? In Scala get* is not popular, and idiomatic way is do just `def order` (especially considering that likely it will be rewritten with a constant in implementation, as group order is a constant indeed). 

The same for all other get* methods in this class

Reply:
renamed